src/backend/dx11/src/conv.rs

```diff
@@ -79,3 +82,23 @@ pub(crate) fn map_format(format: Format) -> Option<DXGI_FORMAT> {
 
     Some(format)
 }
+
+pub(crate) fn map_viewport(viewport: &Viewport) -> D3D11_VIEWPORT {
```

Comment:
why `pub(crate)` instead of just `pub`? `conv` should be a private module anyway imo.